openmls/src/group/core_group/test_proposals.rs

```diff
@@ -447,7 +447,7 @@ fn test_group_context_extensions(
     .expect("Error joining group.");
 }
 
-#[apply(ciphersuites_and_backends)]
+//#[apply(ciphersuites_and_backends)]
```

Comment:
This seems like a typo?

Reply:
Good catch! It kind of is a typo. I regularly comment out these macro invocations so `rust-analyzer` is a little more helpful when working on tests. I usually try not to commit these, but it seems like this one slipped through. Then again, this is not meant to be merged just yet.

Reply:
Oops, I didn't notice the `draft` status. 😬 Yup, ++rust-analyzer.